recipes/django-filer/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "django-filer" %}
+{% set version = "1.3.0" %}
+{% set sha256 = "694fed2d8a3468ab303c1922215c5007f8b60fd4893063038e2fedf685241030" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - django >=1.8
```

Comment:
This differs from what's in setup.py

Reply:
@sannykr, can you please address this issue before asking for another review

Reply:
@isuruf: Sorry about that. I just updated the django-version.